processing/src/test/java/org/apache/druid/query/rowsandcols/semantic/FramedOnHeapAggregatableTest.java

```diff
@@ -371,7 +370,7 @@ public void testUnboundedWindowedAggregation()
     FramedOnHeapAggregatable agger = FramedOnHeapAggregatable.fromRAC(rac);
 
     final RowsAndColumns results = agger.aggregateAll(
-        new WindowFrame(WindowFrame.PeerType.ROWS, true, 0, true, 0, null),
+        WindowFrame.rows(null, null),
```

Comment:
what is the difference between `WindowFrame.rows(null, 0)` and `WindowFrame.unbounded()`? Is it just the rows frame unwraps as `OffsetFrame`? can we use them interchangeably?

Reply:
`WindowFrame.unbounded()` is the same as `WindowFrame.rows(null,null)` ; during compilation we will do `unbounded` in those cases in `Windowing`

changeds this call to `unbounded()`